### ISSUE TITLE
The denote D in paper

### ISSUE THREAD

**Issue by zf223669** [zalandoresearch/pytorch-ts#98]
Dear Kashif Rasul:
       I am reading your paper :” MULTIVARIATE PROBABILISTIC TIME SERIES FORE[1]CASTING VIA CONDITIONED NORMALIZING FLOWS”.
Now I have a confusion, I would like to consult you:
       You denote the entities of multivariate time series by xit ∈ R for I ∈{1，…,D} on page 4. I though the dimension D was represent the feature of the x at time t. However, on page5, you said:“Given D, a batch of time series….. ”And “In practice, the time series x1:T in a batch D are selected…”does the D represent the num of a batch?
   It seem that the D on page 4 was not the same meaning of the D on page 5?

       Could you explain more detail about the D for me?
Thanks a lot.

**Comment by kashif** [comment]
right one is cal{D} which is my batch and the size of this set is the batch_size. 

The other capital D is the multivariate dim of the time series in question... i.e. the size of the target vector at each time point.

To this vector we also add covariates meaning the final vector at each time point ends up being typically bigger than D but, regardless the task is to predict the distribution of the next time point which is a vector of size D 

The way we train is to sample context and prediction sized windows from our time series in order to populate the batch \cal{D}

hopefully that clears up the confusion?

**Comment by zf223669** [comment]
Thank you for your reply.
It is say the two capital D are difference? However,
Why not use different capital?

**Comment by kashif** [comment]
yes the two symbols are different one is $\cal{D}$ my batch and the other is just capital $D$ the multivariate dim. I will remember to use different symbols next time!

**Comment by zf223669** [comment]
Thanks.
